cmd/consensus/main.go

```diff
@@ -98,7 +98,7 @@ func main() {
 			flags.UintVar(&guaranteeLimit, "guarantee-limit", 1000, "maximum number of guarantees in the memory pool")
 			flags.UintVar(&resultLimit, "result-limit", 10000, "maximum number of execution results in the memory pool")
 			flags.UintVar(&approvalLimit, "approval-limit", 1000, "maximum number of result approvals in the memory pool")
-			flags.UintVar(&sealLimit, "seal-limit", 10000, "maximum number of block seals in the memory pool")
+			flags.UintVar(&sealLimit, "seal-limit", 44200, "maximum number of block seals in the memory pool")
```

Comment:
```suggestion
			// the default value is able to buffer as many seals as would be generated over ~12 hours. In case it
			// ever gets full, the node will simply crash instead of employing complex ejection logic.
			flags.UintVar(&sealLimit, "seal-limit", 44200, "maximum number of block seals in the memory pool")
```